src/cli/filter_arg.zig

```diff
@@ -0,0 +1,302 @@
+const std = @import("std");
+const root = @import("root");
+const bun = root.bun;
+const string = bun.string;
+const Output = bun.Output;
+const Global = bun.Global;
+const strings = bun.strings;
+const json_parser = bun.JSON;
+const Glob = @import("../glob.zig");
+
+const Package = @import("../install/lockfile.zig").Package;
+
+fn globIgnoreFn(val: []const u8) bool {
+    if (val.len == 0) {
+        return false;
+    }
+    // skip hidden directories
+    if (val[0] == '.') {
+        return true;
+    }
+    // skip node_modules
+    if (strings.eqlComptime(val, "node_modules")) {
```

Comment:
we could skip `CMakeFiles` as well, but it's less likely to exist so probably not necessary

Reply:
we should also skip`.git`

Reply:
but that can be added later